Dockerfile.sdk

```diff
@@ -129,7 +132,48 @@ RUN cmake -DCMAKE_INSTALL_PREFIX=/workspace/install \
 RUN make -j16 cc-clients python-clients java-clients && \
     rm -fr ~/.m2
 
+############################################################################
+## Create sdk container
+############################################################################
+FROM ${BASE_IMAGE}
+
+# Ensure apt-get won't prompt for selecting options
+ENV DEBIAN_FRONTEND=noninteractive
+
+# DCGM version to install for Model Analyzer
+ARG DCGM_VERSION=2.2.9
```

Comment:
Nit: Should we add DCGM_VERSION at the top as well? Can --build-arg propagate through stages for docker build?

Reply:
looks good otherwise

Reply:
Right, I think we should set DCGM_VERSION as a global arg so that --build-arg can be passed through each build stage. Thanks for the catch!

Reply:
You can define the DCGM arg with the group of args in L42 like 

```
ARG DCGM_VERSION=2.2.9
``` 

But inside the final stage for creating the sdk container you will need to add it again

```
ARG DCGM_VERSION
```

For example refer to TRITON_CORE_REPO_TAG